spec/features/admin_user_create_events_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require 'capybara/rspec'
+require_relative '../support/features/admin_user_helpers'
```

Comment:
Require all support files withing your `spec_helper.rb` like this
```ruby
Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
```

Reply:
Maybe you meant rails_helpers. In spec_helpers, he does not see it, but in rails_helpers everything is fine.